### PR TITLE
Modify URL's to be INAV specific

### PR DESCRIPTION
**Pull request by dan557**
Extension of #282 
Change URL's to be INAV specific
Remove any sections that INAV doesn't have such as Travis-Ci and YouTube

### CODE REVIEW

**Comment by skaman82** [comment]
@digitalentity There are some public INAV webpage files. What is the URL for that website?

**Comment by DzikuVx** [comment]
@skaman82 that would be my making of http://inav.shtr.eu

**Comment by skaman82** [comment]
why such a cryptic url?

**Comment by DzikuVx** [comment]
Costed null 😉

File: README.md [hunk 1]
```diff
@@ -42,47 +42,36 @@ The MultiWii software, from which baseflight originated, violates many good soft
...
+There is lots of documentation here: https://github.com/iNavFlight/inav/tree/master/docs
 
 If what you need is not covered then refer to the baseflight documentation. If you still can't find what you need then visit the #cleanflight on the Freenode IRC network
 
 ## IRC Support and Developers Channel
 
 There's a dedicated IRC channel here:
 
-irc://irc.freenode.net/#cleanflight
+irc://irc.freenode.net/#inavflight
```
Comment (gaelj) [inline README.md]: broken link. update to https://gitter.im/iNavFlight/inav

File: README.md [hunk 2]
```diff
@@ -42,47 +42,36 @@ The MultiWii software, from which baseflight originated, violates many good soft
...
 
 If you are using windows and don't have an IRC client installed then take a look at HydraIRC - here: http://hydrairc.com/
```
Comment (gaelj) [inline README.md]: Blank page

File: README.md [hunk 3]
```diff
@@ -96,20 +85,14 @@ Contributions are welcome and encouraged.  You can contribute in many ways:
...
 ## Developers
 
-Please refer to the development section in the [docs/development](https://github.com/cleanflight/cleanflight/tree/master/docs/development) folder.
-
-TravisCI is used to run automatic builds
-
-https://travis-ci.org/cleanflight/cleanflight
-
-[![Build Status](https://travis-ci.org/cleanflight/cleanflight.svg?branch=master)](https://travis-ci.org/cleanflight/cleanflight)
+Please refer to the development section in the [docs/development](https://github.com/iNavFlight/inav-configurator/tree/master/docs/development) folder.
```
Comment (gaelj) [inline README.md]: Replace by https://github.com/iNavFlight/inav/tree/master/docs/development

Comment (stronnag) [inline README.md]: 1. It should not be necessary for us to pimp Hydra's irc windows's client 
2. perhaps we should mention the RCG thread, as the irc channel is not exactly a hub of activity

**Comment by gaelj** [comment]
@dan557 for the configurator tool, maybe just comment out the section, as I'm sure it will make it's way into the google store.
Comment in markdown should be `[//]: #` (http://stackoverflow.com/questions/4823468/comments-in-markdown)

**Comment by dan557** [comment]
I will get those changes complete within the next couple days

**Comment by gaelj** [comment]
Can you fix the conflicts @dan557 ? (you need to rebase on current master)

File: README.md [hunk 4]
```diff
@@ -42,47 +44,38 @@ The MultiWii software, from which baseflight originated, violates many good soft
...
 
-There is lots of documentation here: https://github.com/cleanflight/cleanflight/tree/master/docs
+There is lots of documentation here: https://github.com/iNavFlight/inav/tree/master/docs
 
-If what you need is not covered then refer to the baseflight documentation. If you still can't find what you need then visit the #cleanflight on the Freenode IRC network
+If what you need is not covered then refer to the baseflight documentation. If you still can't find what you need then visit the #iNavflight on the Freenode IRC network
 
 ## IRC Support and Developers Channel
 
 There's a dedicated IRC channel here:
```
Comment (gaelj) [inline README.md:63]: gitter room instead of irc channel

File: README.md [hunk 5]
```diff
@@ -42,47 +44,38 @@ The MultiWii software, from which baseflight originated, violates many good soft
...
-Please subscribe and '+1' the videos if you find them useful.
+https://gitter.im/iNavFlight/inav
 
 ## Configuration Tool
 
-To configure INAV you should use the INAV-configurator GUI tool (Windows/OSX/Linux) that can be found here:
+To configure INAV you should use the INAV-configurator GUI tool (Windows/OSX/Linux).
 
-https://chrome.google.com/webstore/detail/cleanflight-configurator/enacoimjcgeinfnnnpajinjgmkahmfgb
+Currently you have to download the sourcecode and load into Chrome manually from the github page [//]: # (Can be found here: https://chrome.google.com/webstore/detail/inav-configurator/)
```
Comment (gaelj) [inline README.md:71]: the [//] should be at the start of a new line

File: README.md [hunk 6]
```diff
@@ -42,47 +44,38 @@ The MultiWii software, from which baseflight originated, violates many good soft
...
-See: https://github.com/cleanflight/cleanflight/blob/master/docs/Installation.md
+See: https://github.com/iNavFlight/inav/blob/master/docs/Installation.md
 
 ## Documentation
 
-There is lots of documentation here: https://github.com/cleanflight/cleanflight/tree/master/docs
+There is lots of documentation here: https://github.com/iNavFlight/inav/tree/master/docs
 
-If what you need is not covered then refer to the baseflight documentation. If you still can't find what you need then visit the #cleanflight on the Freenode IRC network
+If what you need is not covered then refer to the baseflight documentation. If you still can't find what you need then visit the #iNavflight on the Freenode IRC network
```
Comment (gaelj) [inline README.md:59]: visit the gitter chat room

**Comment by dan557** [comment]
Continued in #423

**Pull request closed without merging**